.vitepress/theme/components/PreferenceSwitch.vue

```diff
@@ -1,64 +1,34 @@
 <script setup lang="ts">
 import { VTSwitch, VTIconChevronDown } from '@vue/theme'
-import { useRoute } from 'vitepress'
-import { inject, Ref } from 'vue'
-import {
-  preferCompositionKey,
+import { inject } from 'vue'
+import usePreferences, {
   preferComposition,
-  preferSFCKey,
-  preferSFC
-} from './preferences'
-
-const route = useRoute()
-const show = $computed(() =>
-  /^\/(guide|tutorial|examples)\//.test(route.path)
-)
-const showSFC = $computed(() => !/^\/guide/.test(route.path))
-
-let isOpen = $ref(true)
-
-const toggleOpen = () => {
-  isOpen = !isOpen
-}
+  preferSFC,
+} from '../composables/usePreferences'
+
+const {
+  isOpen,
+  shortcutInfo,
+  showPreference,
+  showSFC,
```

Comment:
It seems that `showSFC` is no longer being returned, so the second switch is currently missing on the Tutorial and Examples pages.